pkg/installer/custominstallconfig.go

```diff
@@ -248,3 +368,98 @@ func appendFilesToCvoOverrides(a asset.WritableAsset, g graph.Graph) (err error)
 
 	return nil
 }
+
+func indent(indention int, v string) string {
+	newline := "\n" + strings.Repeat(" ", indention)
+	return strings.Replace(v, "\n", newline, -1)
+}
+
+func appendMdsdFiles(bootstrap *bootstrap.Bootstrap, bootstrapLoggingConfig *bootstraplogging.Config) error {
+	baseEtcPath := "etc"
+	baseSystemdPath := "systemd/units"
+
+	config := struct {
+		LoggingConfig *bootstraplogging.Config
```

Comment:
If this is the only thing we are passing it might be tidier to remove the `LoggingConfig` prefix from all of the variables in the file.